### PR TITLE
CFP-64 - Fix: Thrive changes not saved due to Custom font plugin.

### PR DESCRIPTION
**Pull request by yssambare12**
**Description** - Thrive changes not saved due to Custom font plugin

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -151,6 +151,9 @@ Yes, Custom Fonts is completely free to use, without any limitation.
 
 
 == Changelog ==
+= 2.1.1 =
```
Comment (imnavanath) [inline readme.txt:154]:
```suggestion
= 2.1.2 =
```

File: classes/class-bsf-custom-fonts-render.php [hunk 2]
```diff
@@ -132,9 +132,14 @@ public function __construct() {
 			add_filter( 'fl_builder_font_families_system', array( $this, 'bb_custom_fonts' ) );
 
 			// Add font files style.
-			add_action( 'wp_enqueue_scripts', array( $this, 'preload_styles' ), 1 );
-			add_action( 'init', array( $this, 'add_block_assets_style' ) );
+			// Add font files style.
+			if ( is_plugin_active( 'thrive-apprentice/thrive-apprentice.php' ) || is_plugin_active( 'thrive-product-manager/thrive-product-manager.php' ) ) {
```
Comment (imnavanath) [inline classes/class-bsf-custom-fonts-render.php:136]: Can we use `defined()` condition here, instead of is_plugin_active() ?

Comment (yssambare12) [reply classes/class-bsf-custom-fonts-render.php:136]: Resolved in this PR - https://github.com/brainstormforce/custom-fonts/pull/113

File: readme.txt (hunk 1 above)
Comment (yssambare12) [reply readme.txt:154]: Updated in this - #113

File: classes/class-bsf-custom-fonts-render.php [hunk 3]
```diff
@@ -132,9 +132,14 @@ public function __construct() {
 			add_filter( 'fl_builder_font_families_system', array( $this, 'bb_custom_fonts' ) );
 
 			// Add font files style.
-			add_action( 'wp_enqueue_scripts', array( $this, 'preload_styles' ), 1 );
-			add_action( 'init', array( $this, 'add_block_assets_style' ) );
+			// Add font files style.
```
Comment (imnavanath) [inline classes/class-bsf-custom-fonts-render.php:135]:
```suggestion
```

**Comment by yssambare12** [comment]
Closing this PR as it's continue here - https://github.com/brainstormforce/custom-fonts/pull/113

**Pull request closed without merging**